tensorboard/webapp/notification_center/_views/notification_center_container.ts

```diff
@@ -62,7 +62,9 @@ export class NotificationCenterContainer {
     })
   );
 
-  constructor(private readonly store: Store<State>) {}
+  constructor(private readonly store: Store<State>) {
+    this.store.dispatch(actions.initAction());
```

Comment:
this is actually wrong; you should not dispatch an `initAction` like this: i.e., view is a representation of the state. By reasoning with the state alone, you should know when this component is mounted. What are you trying to achieve here?

Reply:
It is intended to decide if the red dot should show or not. 
The `getItem` is removed out of initialState so I am trigging an action to read the timestamp in data_source
what would be the appropriate flow of this?